azerothcore/azerothcore-wotlk
# fix(DB/SAI): Rallying Cry of the Dragonslayer should happen…

… on quest hand in.



## Changes Proposed:
-  When handing in head of Onyxia (lv 60 q; unavailble without mod-progression-module) or head of nefarian (horde), rallying cry of the dragonslayer should happen and the head on the stick should show


## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11267
- Closes https://github.com/chromiecraft/chromiecraft/issues/3073

## SOURCE:

Vmangos for coordinates.

## Tests Performed:

- Tested


## How to Test the Changes:


1. .q add 7784 as horde
2. hand in and get buff.
3. see the head on the stick
4. notice the two yell lines from the npc

## Known Issues and TODO List:


- [x] Handing in both Head of Onyxia and Head of Nefarian as horde will cause only 1 to happen... -> fixed with conditions and some changes to the smartscript


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
